client/src/pages/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Button, IconButton, CssBaseline, Avatar, Typography, Container, Paper, Box } from '@material-ui/core';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import useStyles from './useStyles';
+
+export default function ProfilePhoto(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" className={classes.main}>
+      <CssBaseline>
```

Comment:
Please remove the `CssBaseline` component here as it is a global reset, you only need to include it once (ideally in the App.tsx)